be/src/storage/rowset/vectorized/segment_iterator.cpp

```diff
@@ -342,6 +342,7 @@ Status SegmentIterator::_init_column_iterators(const Schema& schema) {
             iter_opts.use_page_cache = _opts.use_page_cache;
             iter_opts.rblock = _rblock.get();
             iter_opts.check_dict_encoding = check_dict_enc;
+            iter_opts.reader_type = _opts.reader_type;
```

Comment:
I have a question that has this not been in effect?

Reply:
yes, it is not in effect before. And I modify the logic, it is not effective yes now. I add this just to prevent the future use bug.